utils_test.go

```diff
@@ -97,10 +98,47 @@ func (r With) Path(p string) string {
 	return fmt.Sprintf("%s%s", r, p)
 }
 
+func HandleWs(w http.ResponseWriter, r *http.Request) {
```

Comment:
Move code to handle ws connection to function in order to use it few routes (this code is used for testing purpose)